cs.json

```
@@ -0,0 +1,347 @@
+{
+    "name" : "Čeština",
+    "live" : true,
+    "translations" : {
+        "meta" : {
+            "thForDate" : true,
+            "dateFormat" : "[month] [date] [year]",
```

```suggestion
            "dateFormat" : "[date]. [month] [year]",
```
I looked it up for Czech. It seems to be the same as for the majority of the European languages, so this is probably a more suitable format 🙂